src/main/webapp/app/assessment/assessment-header/assessment-header.component.ts

```diff
@@ -60,6 +61,7 @@ export class AssessmentHeaderComponent {
     // Icons
     faSpinner = faSpinner;
     faSave = faSave;
+    faArrowRight = faSquareCaretRight;
```

Comment:
does not look good and might be confusing or is it on purpose?

Reply:
It was on purpose, because the icon is used for the tooltip indicating that you can press the right arrow on the keyboard. But of course this might be confusing.

Reply:
I know the change was in purpose, but probably not the inconsistency that left and ride side are different? can you refactor (rename) the left side as well? I think the convention is that both sides are the same.
Otherwise, developers who only look into the html are confused why the icon looks different than the name would suggest